### ISSUE TITLE
Release on conda-forge

### ISSUE THREAD

**Issue by mdhaber** [mdhaber/marray#43]
Would be good to get this on conda-forge.

@lucascolley, would you be willing to do that as you did for `scipy_doctest`?

**Comment by lucascolley** [comment]
yep, happy to do this. In case you are interested, the process is just:
- use `grayskull` to generate the recipe. I do this in the #recipe-generator channel of the prefix.dev discord with the command `/grayskull scipy-doctest pypi`
- Make a PR to conda-forge/staged-recipes following the instructions at https://conda-forge.org/docs/maintainer/adding_pkgs/#step-by-step-instructions

**Comment by mdhaber** [comment]
I'll try it. Thanks!

**Comment by mdhaber** [comment]
Hmm "Feedstock with the same name exists in conda-forge."

`marray` is not listed as a package at https://conda-forge.org/packages/. 
I see it here: https://anaconda.org/conda-forge/marray. 

Thoughts on what I should do? Rename the feedstock (maybe `python-marray`?) without changing the name of the package?

**Comment by lucascolley** [comment]
If there is an existing conda-forge::marray package, your best bet is to try to contact the owner of that name, and reach out at https://github.com/conda-forge/admin-requests if that doesn't get anywhere. I wouldn't be too optimistic about taking over the name to be honest, although the last upload was over 7 years ago, so worth a shot...

**Comment by lucascolley** [comment]
the feedstock name can be anything, sure, but the problem is if you want `mamba install marray` to work

**Comment by mdhaber** [comment]
I've seen many packages where the `pip` or `conda`/`mamba` name isn't aligned with what you `import`. (e.g. `pytorch` vs `torch`, right?) It's annoying, but it would be OK with me to name it something different on conda-forge while I'm waiting to hear from  the name owners. Does that make sense?

In that case, suggestion on how to rename it on conda-forge?

**Comment by lucascolley** [comment]
That's true. In which case you just need to set the name you want in the template on the first line of the recipe.

**Comment by mdhaber** [comment]
Also @keewis how should I pronounce your username in my head? (KEE-weez, KEE-wiss?)

**Comment by keewis** [comment]
I literally created my username by pressing random keys on my keyboard and making the result somewhat pronounceable. Thus, there's no fixed pronunciation but I personally would go with the latter.

As for the temporary name on conda-forge, I'd probably go with `python-marray` because it's the most straight-forward (but either is fine, and yes, choosing a temporary name does make sense to me). Either way I'd describe the situation in the `staged-recipes` PR, which would allow the reviewer to give advice on how to best proceed.

**Comment by mdhaber** [comment]
I'll give it a bit of time for the authors to reply there.

**Comment by mdhaber** [comment]
Shall we do another release before the conda forge maintainers get to the PR? See anything that really should happen before we do that?

**Comment by lucascolley** [comment]
Yeah, no harm in doing another release first
